### ISSUE TITLE
Military time not standardized

### ISSUE THREAD

**Issue by mnewberg** [mnewberg/whitmanc.github.io#4]
@whitmanc   Can you please use something like UTC time? My understanding is that the current code does not account for time zones, so it's purely based off of the user's browser's time?

e.g. If I set a store's hours to 900, it will only show for 9am in all time zones, regardless of where I am as a user

**Comment by whitmanc** [comment]
Yep, good catch, that's easy

On Friday, June 26, 2015, mnewberg notifications@github.com wrote:

> Assigned #4 https://github.com/mnewberg/whitmanc.github.io/issues/4 to
> @whitmanc https://github.com/whitmanc.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mnewberg/whitmanc.github.io/issues/4#event-341488810.

## 

[image: Authorly] http://www.authorly.com

Chris Whitman / President & Founder
+1 (424) 274-8320 / chris@authorly.com

http://www.authorly.com

[image: Facebook] http://facebook.com/authorly [image: Twitter]
http://twitter.com/tryauthorly [image: Youtube]
http://www.youtube.com/channel/UCrQoWev_FjCszNBjF_5N1LQ?feature=watch [image:
Linkedin] http://www.linkedin.com/company/authorly

This e-mail message may contain confidential or legally privileged
information and is intended only for the use of the intended recipient(s).
Any unauthorized disclosure, dissemination, distribution, copying or the
taking of any action in reliance on the information herein is prohibited.
E-mails are not secure and cannot be guaranteed to be error free as they
can be intercepted, amended, or contain viruses. Anyone who communicates
with us by e-mail is deemed to have accepted these risks. Authorly is not
responsible for errors or omissions in this message and denies any
responsibility for any damage arising from the use of e-mail. Any opinion
and other statement contained in this message and any attachment are solely
those of the author and do not necessarily represent those of the company.

**Comment by whitmanc** [comment]
Resolved, please confirm @mnewberg

**Comment by mnewberg** [comment]
@whitmanc yeah, it works now thanks. Only question is what to do for cases where the store is open until 9pm EST which is 1am UTC the next day. Will it work with a larger array of times for the day so I can, for instance, put in the initial 0000-1000 value followed by the remaining hours?
